### DISCUSSION TITLE
stream = False , no work.

### DISCUSSION THREAD

**Discussion by WinXsp** [open-webui/open-webui#3445] [General]
# Bug Report

## Description

**Bug Summary:**
stream = False , no work.

**Steps to Reproduce:**
![111](https://github.com/open-webui/open-webui/assets/26323067/ef882070-299c-47e2-9ca5-ccfc7d7b98c8)
![222](https://github.com/open-webui/open-webui/assets/26323067/70241299-2620-4d35-87ad-bf0242d5a1ff)

**Comment by silentoplayz** [comment]
No info = Moved to discussion
